components/user/UserLink.js

```diff
@@ -92,17 +94,18 @@ export default function UserLink({
         </span>
       )}
       {manage && <Bulb isEnabled={isEnabled} />}
+      </div>
     </Link>
   );
 
   const edit = (link) => (
-    <Edit href={`/account/manage/link/${link._id}`} label={`${link.name} Link`}>
+    <Edit className={"mt-5 ml-4"} href={`/account/manage/link/${link._id}`} label={`${link.name} Link`}>
```

Comment:
Could've just put the classname styles into the Edit component instead of passing them as props. Works the same either way.

Reply:
Nope it did not, I tried doing this in the first place however it impacted other places where the Edit component is used, as a result I decided to pass the classNames as props for this case, so they don't impact other places.

Reply:
Ah, ok. Wasn't aware. Will vouch with an approval then.